service/pkg/server/services.go

```diff
@@ -167,8 +167,20 @@ func startServices(ctx context.Context, cfg config.Config, otdf *server.OpenTDFS
 				}
 			}
 
+			svcConfig := cfg.Services[ns]
+			if ns == "kas" {
+				logger.Debug("updating kas key configuration", slog.String("namespace", ns), slog.Any("legacyConfig", cfg.Server.CryptoConfig2024))
+				// Upgrade the the kas configuration, if there is a legacy `CryptoProvider` configuration
+				// present in the otdf server config.
+				if cfg.Server.CryptoConfig2024 != nil {
```

Comment:
`cfg.Server.CryptoConfig2024` is this the legacy config renamed.  should the new config be named `cfg.Server.CryptoConfig2025`.  I have also seen append `_V1` 
is this versioned in the yaml?

Reply:
There is no new config under `Server`; after this it is moved entirely to the `services.kas` field.

The type names are not exposed in the YAML config file, and the names of the fields can/will be overridden with tags (e.g. `mapstructure:"config2025") as needed. The config struct types and their fields need to be exported (capitalized) for use with the `mapstructure` library, but we have as of yet not treated them as part of the API and subject to standard versioning considerations. However, we may want to do that before marking this whole library as 1.x.

I did a grep through our internal code and was unable to find references to the moved/removed/renamed types